monitoring/mongodb/alerts.yaml

```
@@ -139,3 +139,24 @@ groups:
     annotations:
       description: 'MongoDb has low disk space'
       summary: 'MongoDb has low disk space'
+  
+  - alert: MongoDbNodeRecovering
+    expr: |
+      sum by (pod) (mongodb_rs_members_state{namespace="${namespace}", pod=~"${service}.*", rs_state="3"} >= 1)
+    for: 5m
+    labels:
+      severity: warning
+    annotations:
+      description: "Mongodb node in `{{ $labels.pod }}` is in the 'recovering' state for more than 10 minutes."
+      summary: MongoDB node recovering for too long
+
+  - alert: MongoDbNodeNotSynced
+    expr: |
+      sum by (pod) (mongodb_mongod_replset_number_of_members{set="data-db-mongodb-sharded-shard-0"}) != ${replicas}
+    for: 10m
+    labels:
+      severity: critical
+    annotations:
+      description: "MongoDB node in `{{ $labels.pod }}` is not present in the replica set anymore."
```

```suggestion
      description: "The MongoDB instance `{{ $labels.pod }}` is out of the replica set. It does not longer receive any data and must be added back to the cluster to avoid performance and storage problems."
```

Just a note: if we add new alerts, we also want to improve our product doc, so the user can quickly find that, for this example, the right procedure we suggest is to perform a full init sync.